cytosnake/cli/exec/workflow_exec.py

```diff
@@ -61,7 +61,6 @@ def __base_exec(
         unlock=unlock,
         forceall=force,
         use_conda=True,
-        conda_prefix=env_manager,
```

Comment:
Would it make sense to open this parameter up to a CytoSnake user (if it isn't already)? Here I'm wondering whether one might want to distinguish between Snakemake defaults for the `conda_prefix` (mostly in external compute environments).

Reply:
The `configurational.yaml` contains parameters on how to execute workflows. However, there's no set structure yet. That's a good point. I'll make this as a separate PR.

Reply:
Created issue #57